### DISCUSSION TITLE
Feature Request: Have stats include visits to country subdivisions, i.e.states/provinces

### DISCUSSION THREAD

**Discussion by dunc1ca** [Freika/dawarich#372] [Ideas]
Stats could include states or provinces visited within each country.  Also, after selecting a specific year, show the specific countries/cities/provinces visited that year.

**Comment by Freika** [comment]
Good idea!

**Comment by deliantwo** [comment]
I don't know if it's a separate feature, but it would be nice to be able to filter points by country.

The immediate use case for me is to find some outliers, some "definitely wrong points" and delete them.

**Comment by erickkist-cpu** [comment]
I would like this a lot. As adition, it would great when it is possible to view it in a tree-like structure. As an example The Netherlands. On the left side of the screen there is the word "The Netherlands". On the right of it, connected with lines to the word "The Netherlands", all the provinces that is registred and again on the right thereof, also connected with the lines to the specific province, all the cities that are registred.

**Comment by LubosBa** [comment]
I am topping this as well, I would like to see it also implemented on the Scratch map view. I mostly use Dawarich to track the trips I've done with my girlfriend and we mostly just go to Italy, we love scratch maps but the country scratching is just too bug, we would love to see it on a regional / province level.

**Comment by tenjin43** [comment]
I would also like to see this feature.
For me, it would be especially useful if Stats could show visited cities in a hierarchy such as:

Country > administrative division / region > city

Examples:

### 🇯🇵 Japan / 日本
Japan
- Tokyo / 東京都
  - Tokyo
  - Hachioji
  - Machida
- Osaka / 大阪府
  - Osaka
  - Sakai
  - Suita
- Kyoto / 京都府
  - Kyoto
  - Uji
  - Maizuru
- Toyama / 富山県
  - Toyama
  - Takaoka
  - Imizu

### 🇺🇸 United States
United States
- California
  - Los Angeles
  - San Francisco
  - San Diego
- Nevada
  - Las Vegas
  - Reno
- New York
  - New York
  - Buffalo

### 🇬🇧 United Kingdom
United Kingdom
- England
  - London
  - Manchester
  - York
- Scotland
  - Edinburgh
  - Glasgow
- Wales
  - Cardiff
  - Swansea
- Northern Ireland
  - Belfast

### 🇰🇷 South Korea / 韓国
South Korea
- Seoul / ソウル特別市
  - Seoul
- Busan / 釜山広域市
  - Busan
- Gyeonggi-do / 京畿道
  - Suwon
  - Seongnam
  - Goyang
  - Yongin

The current flat city list becomes difficult to browse when many cities are visited in the same country.
A hierarchical view would make the Stats page much easier to understand, especially for countries where regions, prefectures, states, provinces, or metropolitan cities are important.
This would be useful for examples such as:
- Japan / 日本: prefecture / 都道府県 > city
- United States: state > city
- United Kingdom: country / region > city
- South Korea / 韓国: province / metropolitan city > city

It would also be fine if this is optional, for example:
- Flat city list
- Group by administrative division
